sstables/sstables.cc

```diff
@@ -3638,6 +3638,9 @@ future<data_sink> file_io_extension::wrap_sink(const sstable& sst, component_typ
     co_return co_await make_file_data_sink(std::move(f), file_output_stream_options{});
 }
 
+future<data_source> file_io_extension::wrap_source(const sstable& sst, component_type c, data_source source) {
+    co_return source;
```

Comment:
Is that really enough? The file_io_extention::wrap_sink() tries to elaborate on whether the sink should be wrapped with the help of a noop-file and only returns the input sink if it doesn't. Shouldn't it be the same here?

Reply:
I dont think this noop-file thingy is needed here, at least, I dont see any reason to apply it here